### ISSUE TITLE
Build a new website using Docusaurus

### ISSUE THREAD

**Issue by tdurand** [opendatacam/opendatacam-website#21]
For v4 release, want to improve the website to include the documentation on it so newcomers don't necessarly have to go to github to see what OpenDataCam is about

I benchmarked a few tools and I think https://docusaurus.io/ seems the most pragmatical choice.

Will get started on this soon ✌

**Comment by tdurand** [comment]
Finaly got started today, pushed on v4 branch, very wip yet..

quick notes for myself, for styling:

https://docusaurus.io/docs/styling-layout

using this homemade css framework: https://infima.dev/docs/layout/spacing

**Comment by napking** [comment]
Hi

I'm going through a fresh install of my jetson nano and figured I would review the install documentation at the same time. I wouldn't consider myself super tech savvy, but enough to get my own device up a running.

I'm editing the FLASH_JETSON.md and JETSON_NANO.md in my own fork right now. I'm updating the language to be a bit friendlier and ensuring that the order of operations is a bit more linear to follow. Is this something that would be valuable to contribute?

Also, is OpenDataCam still recommending jetpack 4.3? I've just read through the linked issues over on the NVIDIA forum and some users suggest v4.5.1 still persists in slow down.

**Comment by tdurand** [comment]
hi ! 

> I'm editing the FLASH_JETSON.md and JETSON_NANO.md in my own fork right now. I'm updating the language to be a bit friendlier and ensuring that the order of operations is a bit more linear to follow. Is this something that would be valuable to contribute?

yes all feedback is good to take, I will rewrite a bit myself also but if you have already done some you can open a pull request on the main repo titled "Documentation Update" so I see the changes.. if you don't know how to do that you can also just send me a link to the modified files and I will see your changes

> Also, is OpenDataCam still recommending jetpack 4.3? I've just read through the linked issues over on the NVIDIA forum and some users suggest v4.5.1 still persists in slow down.

yes for v3 it is.. but if you use the docker version , latest 4.5.1 should work fine.. I'm in the process of upgrading everything so you can use opendatacam v4 with latest jetpack.. I got a bit delay but expect this by the end of july (hopefuly !)

**Comment by napking** [comment]
> > Also, is OpenDataCam still recommending jetpack 4.3? I've just read through the linked issues over on the NVIDIA forum and some users suggest v4.5.1 still persists in slow down.
> 
> yes for v3 it is.. but if you use the docker version , latest 4.5.1 should work fine.. I'm in the process of upgrading everything so you can use opendatacam v4 with latest jetpack.. I got a bit delay but expect this by the end of july (hopefuly !)

Thanks. Many of my documentation updates were in reference to properly using jetpack 4.3. If that version won't be necessary  for v4 of ODC then I won't bother clarifying that language. I'll send over once I get through my re-build.

**Comment by tdurand** [comment]
> Thanks. Many of my documentation updates were updates to properly using jetpack 4.3. If that version won't be necessary for v4 of ODC then I won't bother clarifying that language

Yeah I guess this won't be necessary because it will be outdated when we release OpenDataCam v4 😁

**Comment by napking** [comment]
I'm still new to using Git with collaborators. I tried sending a pull of my development branch but it was including more changes to my personal branch than I wanted.
Here's my documentation updates:
[https://github.com/napking/opendatacam/tree/development/documentation/jetson](https://github.com/napking/opendatacam/tree/development/documentation/jetson)

**Comment by vsaw** [comment]
I started working on this on the weekend. First step is getting the existing website running on Docusaurus. 

Progress is good, I already
Had it working in Development until I hit this error when doing my first deployment to GitHub https://github.com/vimeo/player.js/issues/32

I hope that the suggested workaround still works. 😅
